backend/loernwerkServer.ts

```diff
@@ -28,10 +35,27 @@ class loernwerkServer {
         app.use(json());
         app.use(urlencoded({ extended: true }));
 
+        // Setting up cookies
+        app.use(
+            session({
+                name: 'loernwerk.session',
+                resave: false,
+                saveUninitialized: false,
+                secret: randomBytes(64).toString('hex'),
+                cookie: {
+                    maxAge: 1000 * 60 * 60 * 24,
```

Comment:
add a comment as to what time frame this is

Reply:
true, 24h is default session length because of teachers workflows